src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1586,6 +1589,22 @@ private boolean bootstrap(final Collection<Token> tokens)
             SystemKeyspace.resetAvailableRanges();
         }
 
+        try
+        {
+            setMode(Mode.WAITING_TO_REQUEST_STREAMS, "Awaiting call to proceed with requesting streams during bootstrap", true);
```

Comment:
Nitty nit: `WAITING_TO_REQUEST_BOOTSTRAP_STREAMS` or sth similar? Since theoretically there are streams in other pieces of the code